commons-dlp/src/main/java/org/exoplatform/commons/dlp/service/impl/DlpPositiveItemServiceImpl.java

```diff
@@ -44,4 +51,26 @@ public void deleteDlpPositiveItem(Long itemId) {
     public DlpPositiveItemEntity getDlpPositiveItemByReference(String itemReference) {
         return dlpPositiveItemDAO.findDlpPositiveItemByReference(itemReference);
```

Comment:
All function of the service should return a DTOObject (in your case DlpPositiveItem
The object DlpPositiveItemEntity is reserved to be used between the service and the DAO.

Reply:
updated !